### PR TITLE
Fix e2e local

### PR DESCRIPTION
**Pull request by sbose78**
- rename repetitive code which caused compile time issues in the end
- add runtime env var which was used to determine `if Darwin`

### CODE REVIEW

File: test/e2e/component_test.go [hunk 1]
```diff
@@ -18,13 +17,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-const (
-	retryInterval        = time.Second * 5
```
Comment (baijum) [inline test/e2e/component_test.go]: I don't see these constants defined anywhere else. And it is used in `TestComponent`.

Comment (sbose78) [reply test/e2e/component_test.go]: https://github.com/redhat-developer/devconsole-operator/blob/master/test/e2e/custom_resource_operator_test.go#L25

Comment (baijum) [reply test/e2e/component_test.go]: That file was removed in Avni's PR #33 But looks like it got introduced again from PR #55

Comment (baijum) [reply test/e2e/component_test.go]: We should remove that file, and just keep `component_test.go`. It is the same test.

Comment (tinakurian) [reply test/e2e/component_test.go]: Fixed

Comment (sbose78) [reply test/e2e/component_test.go]: AH yes!

**Pull request merged**